Automattic/jetpack
# Stats: increase timeout to 20s

## Proposed changes:

The All time top posts API requires ~10s to finish, and is causing timeout errors on Odyssey. We increase the timeout to 20s now, and will revisit the issue to see how we should move forward - improve performance or remove it.

### Other information:


## Jetpack product discussion
pejTkB-Vu-p2

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* Add `sleep( 10 );` at L233 of fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qfgngf%2Qgbc%2Qcbfgf%2Qi1%2Q1%2Qraqcbvag.cuc%3Se%3Q37oq57oo%23233-og
* Sandbox your Jetpack installation `define( 'JETPACK__SANDBOX_DOMAIN', 'xxx.dev.dfw.xxx.com' );`
* Open `/wp-admin/admin.php?page=stats#!/stats/day/posts/site?startDate=2023-10-11&summarize=1&num=-1`
* Ensure All time view works okay

<img width="1194" alt="image" src="https://github.com/Automattic/jetpack/assets/1425433/98d0307c-dd8d-445f-95bc-a212aaa27153">
